pytree.py

```diff
@@ -1,11 +1,55 @@
 #!/usr/bin/env python3
-import subprocess
 import sys
+import os
+import re
 
 
-# YOUR CODE GOES here
+def sort_key(s):
+    return re.sub('[^A-Za-z]+', '', s).lower()
 
 
+def walk_dir(items, nextpath, count, indent, i):
+    if os.path.isdir(nextpath):
+        count[0] += 1
+        indent = indent + '    ' if i == len(items) - 1 else indent + '│   '
+        tree(nextpath, count, indent)
+        indent = indent[:-4]
+    else:
+        count[1] += 1
+
+
+def tree(path, count, indent=''):
+    items = os.listdir(path)
```

Comment:
Oh, I was just wondering if I could use os.walk, so I decided to do it the basic way. Good to know, thanks!